app/models/member.rb

```diff
@@ -75,6 +78,19 @@ def last_name
     name.split.last
   end
 
+  # Force the phone to be only numbers on input
+  def phone=(val)
+    write_attribute(:phone, Phonelib.parse(val).sanitized)
+  end
+
+  # If a phone record exists, return an international form of the number.
+  def format_phone
+    if phone
+      phone = Phonelib.parse(self.phone)
+      phone.international
```

Comment:
Can we use `phone.national` here? Leads to prettier results when displaying the numbers